develop/sounds/custom.md

```diff
@@ -0,0 +1,63 @@
+---
+title: Create Custom Sounds
+description: Learn how to add and use a new sound with the registry.
+authors:
+  - JR1811
+---
+
+# Creating Custom Sounds
+
+## Preparing the audio file
+
+Your audio files need to be formatted in a specific way. OGG Vorbis is an open container format for multimedia data, such as audio, and is used in case of Minecraft's sound files. To avoid problems with how Minecraft handles distance of sound the audio data needs to have only a single channel (Mono).
```

Comment:
```suggestion
Your audio files need to be formatted in a specific way. OGG Vorbis is an open container format for multimedia data, such as audio, and is used in case of Minecraft's sound files. To avoid problems with how Minecraft handles sound distancing, your audio needs to have only a single channel (Mono).
```